python/lsst/drp/tasks/forcedPhotCoadd.py

```diff
@@ -320,6 +320,12 @@ def run(self, measCat, exposure, refCat, refWcs, exposureId=None):
                 Catalog of forced measurement results
                 (`lsst.afw.table.SourceCatalog`).
         """
+        # We want to cache repeated PSF evaluations at the same point coming
+        # from different measurement plugins.  We assume each algorithm tries
+        # to evaluate the PSF twice, which is more than enough since many don't
+        # evaluate it at all, and there's no *good* reason for any algorithm to
+        # evaluate it more than once.
+        exposure.psf.setCacheCapacity(2 * len(self.config.measurement.plugins.names))
```

Comment:
Is the rationale for 2 here is to cache the resulting image from calling `computeImage` and `computeKernelImage`?

Reply:
No, those are already separate caches, so they effectively already have that factor of two.  I'm just being defensive against plugins doing more PSF evaluations than they probably should.

Reply:
Ah, yes. Right!